src/pages/Governance/styles.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from 'react'
```

Comment:
I'd write this card a bit differently to have 100% reusability in the farming interface. Why not having `currency1` and `currency2` props through which to determine if we have to use a `CurrencyLogo` or `DoubleCurrencyLogo` to display asset icons (and in case only `currency1` is passed, we would render the logo and asset name on the same row), and a sort of `body` prop that would be stylized accordingly and contains information about either the governance asset or the farming associated with a pair.

Looking at the mocks it might be a bit hard to make everything work, since dimensions are a bit different from one screen to the other, but I think we can discuss this with @zamli, and reach a bit more consistency, which would help us improving code reusability.